CONTRIBUTING.md

```diff
@@ -240,6 +241,10 @@ Please feel free to add to or edit these sections.
 
 ### Formatting tips
 
-We use markdown for the book formatting.
+We use [markdown](https://daringfireball.net/projects/markdown/) for the book's
+formatting.
```

Comment:
Gruber's markdown is ill-defined, incomplete, and unmaintained. It would be more accurate to link [Pandoc markdown](https://pandoc.org/MANUAL.html#pandocs-markdown) with [Bookdown extensions](https://bookdown.org/yihui/bookdown/markdown-extensions-by-bookdown.html).

Reply:
Is this PR jus waiting for the change of link?

Reply:
It's waiting for me to loop back to it. Will add the link and text in a few ;-)